### DISCUSSION TITLE
The DataGrid example on the website needs an example code on how to show data from a database

### DISCUSSION THREAD

**Discussion by kephax** [microsoft/fluentui-blazor#2413] [Ideas]
This page does not show you how to display data from a database, it would be very useful to have such an example:

https://www.fluentui-blazor.net/DataGrid

**Comment by vnbaaij** [comment]
Showing data from a database ia not requiring anything specific from the library's perspective and isn't any different from what we are already showing in the examples that are there.

It is not the purpose or intent of the demo site to show general Blazor and/or .NET coding examples.

If you would like to help the community and have such an example, perhaps consider publishing a blog post about it?
